api/Node.json

```diff
@@ -1439,10 +1439,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Node/nodePrincipal says this was a Firefox-specific property, but it was never supported?

Reply:
I've removed it in https://github.com/mdn/browser-compat-data/pull/7521.